src/dotnet/Audio.UI.Blazor/Components/AudioRecorder/AudioRecorderToggle.razor

```diff
@@ -5,11 +5,13 @@
     var isRecording = State.ValueOrDefault;
 }
 
-<button class="audio-recorder-toggle p-1" @onclick="_ => ToggleRecording()" >
-    <svg class="fill-current w-8 h-8 @(isRecording ? "recording" : "not-recording")" xmlns="http://www.w3.org/2000/svg" viewBox="-4 -4 32 32">
-        <path d="M16 11c0 2.209-1.791 4-4 4s-4-1.791-4-4v-7c0-2.209 1.791-4 4-4s4 1.791 4 4v7zm4-2v2c0 4.418-3.582 8-8 8s-8-3.582-8-8v-2h2v2c0 3.309 2.691 6 6 6s6-2.691 6-6v-2h2zm-7 13v-2h-2v2h-4v2h10v-2h-4z"/>
-    </svg>
-</button>
+<ButtonSquare Class="audio-recorder-toggle" OnClick="_ => ToggleRecording()">
+    <ChildContent>
```

Comment:
Top secret info: parameter named `ChildContent` with `RenderFragment` type is mapped to component's child content automatically.